### DISCUSSION TITLE
Why I see `React Router Future Flag Warning` in production build?

### DISCUSSION THREAD

**Discussion by rakleed** [remix-run/react-router#12451] [Q&A] [answered]
I updated to react-router@6.28.0, and now I see `React Router Future Flag Warning` console warnings even in a production build. But why? It seems to me that this information should only be displayed in development mode.

**Comment by VillePakarinenPosti** [comment]
Related to this see: https://github.com/remix-run/react-router/pull/12441

**Comment by rakleed** [comment, marked as answer]
Fixed in https://github.com/remix-run/react-router/releases/tag/react-router@6.29.0
